README.md

```diff
@@ -26,13 +26,13 @@ How can I run Sipa?
 -------------------
 
 As a general note, you should have `docker` and `docker-compose`
-installed.  Just using python and e.g. a virtualenv is possible, but
-discouraged.
+installed (or similar software like `podman` and `podman-compose`).  
```

Comment:
Like we discussed, these README changes mentioning podman everywhere do more harm than good, so we should get rid of them. Maybe instead add a small paragraph stating that sipa is also compatible with podman at the end of the `Running on Docker` section?